### PR TITLE
[author] Add d3-annotation@1.9.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by susielu**
Pull request for issue: #10884
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/susielu/d3-annotation
 * Official website (optional, not the repository): http://d3-annotation.susielu.com/
 * NPM package url (optional): https://www.npmjs.com/package/d3-svg-annotation
 * License and its reference: Apache 2.0 https://github.com/susielu/d3-annotation/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 15
   - Count of stars: 250
   - Count of forks: 9
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 47
   - Downloads in the last month: 532

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR, will link to issue
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by susielu** [comment]
#10884 referenced issue @con-ssc @sufuf3

**Comment by sufuf3** [comment]
@cdnjs/intern3 please help review this PR, thanks.

**Comment by sufuf3** [comment]
@susielu you are the author of this library, and I saw there is v1.0.0 in npm (v0.0.0 in npm I think it's not necessary to host).
Do you want to add v1.0.0? 
Which auto-update do you like, npm or GitHub?

**Comment by susielu** [comment]
No need to add v1.0.0, we can just add the latest version, it's the most
stable. Let's do npm autoupdate.

**Comment by sufuf3** [comment]
@susielu I think that if cdnjs no need to host v1.0.0 and v0.0.0. 
Maybe we could just use GitHub auto-update, this will automatically add the old versions from GitHub.
If you have any suggestions, please let me know.

cc @hare1039

File: ajax/libs/d3-annotation/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "homepage": "http://d3-annotation.susielu.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/susielu/d3-annotation.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "d3-annotation.js",
+          "d3-annotation.min.js"
```
Comment (extend1994) [inline ajax/libs/d3-annotation/package.json]: Don't we need `d3-annotation.css`?

**Comment by susielu** [comment]
Sounds good, we can go with git autoupdate. And we can include the css file.

File: ajax/libs/d3-annotation/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/d3-annotation/package.json]: @susielu maybe we can just use something like `d3-annotation*` or `d3-annotation*.+(css|js|map)`?

**Comment by sufuf3** [comment]
@cdnjs/intern3 please help review this PR, thanks.

**Comment by PeterDaveHello** [comment]
Thanks all ✨

**Pull request merged**